server/segment_rpc_test.go

```diff
@@ -809,6 +809,7 @@ func TestSubmitSegment_GenPaymentError_ValidatePriceError(t *testing.T) {
 	balance.On("StageUpdate", mock.Anything, mock.Anything).Return(1, nil, nil)
 	sender := &pm.MockSender{}
 	sender.On("EV", mock.Anything).Return(big.NewRat(1, 1), nil)
+	balance.On("Credit", mock.Anything)
```

Comment:
Can we check that `Credit()` is called with the existing credit (i.e. B's reserved balance) as we do in `TestSubmitSegment_GenPaymentError_CreateTicketBatchError()`?

Reply:
Fixed, will await your comment for another rebase.